wp-content/themes/pub/wporg-learn-2024/patterns/sensei-quiz-notices.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Title: Sensei Quiz Notices
+ * Slug: wporg-learn-2024/sensei-quiz-notices
+ * Inserter: no
+ *
+ * @package wporg-learn-2024
+ */
+
+$lesson_id         = Sensei()->quiz->get_lesson_id();
+$course_id         = Sensei()->lesson->get_course_id( $lesson_id );
+$is_learning_mode  = Sensei_Course_Theme_Option::has_learning_mode_enabled( $course_id );
+$is_awaiting_grade = Sensei_Quiz::is_quiz_awaiting_grade_for_user( $lesson_id, get_current_user_id() );
+$is_pending_grade = $is_learning_mode && $is_awaiting_grade;
+$referrer = esc_url( $_SERVER['HTTP_REFERER'] );
```

Comment:
Did you mean to use this on the next line?

Reply:
That's for testing, thanks 👍 
0437c22